### ISSUE TITLE
Spectral norm + equalized learning rate?

### ISSUE THREAD

**Issue by capilano** [taki0112/StyleGAN-Tensorflow#8]
The original Style Gan implementation does not have spectral norm. I am not sure how spectral norm interacts with the equalized learning rate multiplier. Did you observe improvements by adding spectral norm?

**Comment by rfitz123** [comment]
Also wondering how to implement spectral normalization. I'm having a lot of problems with stability.
